homeassistant/components/frontend/storage.py

```diff
@@ -14,15 +14,29 @@
 from homeassistant.helpers.storage import Store
 from homeassistant.util.hass_dict import HassKey
 
+from .const import SYSTEM_DATA_DEFAULT_PANEL
+
+
+class SystemStorageData(TypedDict, total=False):
+    """System storage data structure."""
+
+    default_panel: str | None
+
+
 DATA_STORAGE: HassKey[dict[str, UserStore]] = HassKey("frontend_storage")
+DATA_SYSTEM_STORAGE: HassKey[SystemStore] = HassKey("frontend_system_storage")
 STORAGE_VERSION_USER_DATA = 1
+STORAGE_VERSION_SYSTEM_DATA = 1
 
 
 async def async_setup_frontend_storage(hass: HomeAssistant) -> None:
     """Set up frontend storage."""
     websocket_api.async_register_command(hass, websocket_set_user_data)
     websocket_api.async_register_command(hass, websocket_get_user_data)
     websocket_api.async_register_command(hass, websocket_subscribe_user_data)
+    websocket_api.async_register_command(hass, websocket_set_system_data)
+    websocket_api.async_register_command(hass, websocket_get_system_data)
```

Comment:
When would we ever want to get it? Shouldn't we just always subscribe to it in frontend. We also have the subscribeOne helper in frontend

Reply:
I copied the get_user_data but I agree, it's not necessary as we can use `susbcribeOne`.